src/main/java/net/mcreator/ui/blockly/CompileNotesPanel.java

```diff
@@ -90,6 +90,10 @@ public List<BlocklyCompileNote> getCompileNotes() {
 		return retval;
 	}
 
+	public void addCompileNote (int index, BlocklyCompileNote note) {

```

Comment:
No need for this. Check how eg. trigger or procedure block API dependencies are checked and how compile notes are shown there. Use existing system for this

Reply:
Is what I did better? I wasn't able to use the same system as blocks, so I decided to use the `updateCompileNotes` method instead.